### DISCUSSION TITLE
Add a New Exchange - Virtual Host Not Visible on Chrome

### DISCUSSION THREAD

**Discussion by jeffreywstevens** [rabbitmq/rabbitmq-server#9169] [Other]
### Describe the bug

On version RabbitmMQ Management Plugin Version 3.11.16. On Windows 11 Chrome, the Virtual Host Drop Down is not visible. On Edge, it is visible.
Chrome:
<img width="485" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/637945/89a0b589-675f-4a2f-82b0-b8011a93ecb8">
Edge:
<img width="329" alt="image" src="https://github.com/rabbitmq/rabbitmq-server/assets/637945/cafa49d2-a017-4906-a7a1-18128daf037b">

### Reproduction steps

1. Use Chrome. No Virtual Host Drop Down.
2. Use Edge: Virtual Host Drop Down Visible.
3.
...

### Expected behavior

Virtual Host is Visible in all browsers.

**Comment by michaelklishin** [comment]
I cannot reproduce.

Besides clearing browser cache, another reason why the dropbox may be hidden is due to different users used. A user that only has access to one virtual host won't see the dropbox IIRC.
